api.md

```diff
@@ -0,0 +1,215 @@
+# CFPB API Standards
+
+This document captures **CFPB's view of API best practices and standards**. We aim to incorporate as many of them as possible into our work.
+
+APIs, like other web applications, vary greatly in implementation and design, depending on the situation and the problem the application is solving.
```

Comment:
Yeah the sentence is kinda proscriptive, but doesn't really finish the thought, `ie`, api's vary -> so don't worry if some of the advice doesn't apply in your case.

Maybe a better reorg of these few sentences would be:
document captures CFPB's view -> but APIs can vary -> so while you should consider your use case -> we'll be aiming to incorporate